_data/navigation.yml

```diff
@@ -1,12 +1,50 @@
-en:
```

Comment:
It's probably fine dropping en at this stage. At some stage we'll want to do a non en version of the page as well, but I don't have any clear ideas what that would entail atm.